kratos/python/add_model_part_to_python.cpp

```diff
@@ -56,11 +56,6 @@ ProcessInfo& GetProcessInfo(ModelPart& rModelPart)
     return rModelPart.GetProcessInfo();
 }
 
-void SetProcessInfo(ModelPart& rModelPart, ProcessInfo& NewProcessInfo)
-{
```

Comment:
why are u removing this? from python i would say it is needed

Reply:
we still need the possibility of creating a copy... the only point is that we will need to create a new pointer with the copy outside (in this case within the wrapper)

Reply:
maybe it's a good opportunity to return the pointer by ref then.
```cpp
const ProcessInfo::Pointer pGetProcessInfo()
```